### PR TITLE
Claims navigation

### PR DESCRIPTION
**Pull request by julia-andersson**
## [APP-XXX]

To fix:
- Push notifications (will be fixed later)
- Custom name on model picker in single item screen
- Changes in store is not observed by view without delay (ex location picker)

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/Journeys/ClaimsProgressBar.swift [hunk 1]
```diff
@@ -5,41 +5,75 @@ import SwiftUI
 import hCore
 import hCoreUI
 
-extension JourneyPresentation {
-    public var addClaimsProgressBar: Self {
-        addConfiguration { presenter in
+extension View {
+
+    /* TODO: MAKE THIS WORK */
+    public var addClaimsProgressBar: some View {
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Journeys/ClaimsProgressBar.swift:11]: @sladan-hedvig not getting updated value or correct height

File: Projects/Claims/Sources/Navigation/ClaimsNavigation.swift [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+        }
+        .detent(
+            presented: $claimsNavigationVm.isBrandPickerPresented,
+            style: .height
+        ) {
+            openBrandPickerScreen()
+                .routerDestination(
+                    for: ClaimFlowItemBrandOptionModel.self
+                ) { brandModel in
+                    /* TODO: FIX - IS NEVER TRIGGERED */
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Navigation/ClaimsNavigation.swift:156]: @sladan-hedvig not getting this to work. model picker is not triggered

**Comment by julia-andersson** [comment]
@sladan-hedvig also, not getting honesty pledge to close after dismissing claims flow

**Comment by sladan-hedvig** [comment]
> @sladan-hedvig also, not getting honesty pledge to close after dismissing claims flow

Fixed + hide honesty pledge when starting claim

**Review by sladan-hedvig — APPROVED** [review]
Did some fixes/changes please check commits.
Othervise looks awesome! 🥇

**Pull request merged**